### ISSUE TITLE
Layout: Tools equivalent to community users

### ISSUE THREAD

**Issue by bwbohl** [music-encoding/music-encoding.github.io#130]

**Comment by musicEnfanthen** [comment]
@bwbohl Could you provide a description of the issue please?

**Comment by bwbohl** [comment]
of course, I don't like the different layouts of theses categories on or website, with emphasis on DIFFERENT…
I was wondering whether to take the same approach as with conferences (which is actually sort of a copy from Community/Projects).

What do you think?

**Comment by musicEnfanthen** [comment]
Yes, would be good to have some "consistency"

**Comment by rettinghaus** [comment]
@bwbohl what's the status of this?

**Comment by bwbohl** [comment]
@rettinghaus so far no work done just at the back of my head ;-)

**Comment by bwbohl** [comment]
At the MEI developer workshop 2024-01 we started to overhaul the tool section; work has progressed and I hope a PR will become available, soon.
